app/views/groups/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<main class="bg-gray-200 h-screen flex flex-col">
+  <header>
+    <nav class="flex items-center justify-between background-blue p-4 text-white">   
+      <a href='/'><i class="fas fa-arrow-left text-2xl"></i></a>
+      <h1 class="text-center font-bold text-2x p-3">TRANSACTIONS</h1>
+      <i class="fa-solid fa-magnifying-glass"></i>
+    </nav>
+  </header>
+  <p style="color: green"><%= notice %></p>
+  <div class="grow">
+    <%= render @group %>
+    <div class="flex justify-end px-3">
+      <%= button_to @group, method: :delete, class: "px-2 py-1 text-2xl rounded text-white bg-red-700" do %>
+        <i class="fa-solid fa-trash-can"></i>
+      <% end %>
+    </div>
+
+    <h2 class="text-center font-bold text-2xl p-3">DETAILS </h2>
+    <div class="flex flex-col justify-between">
+      <div class="flex flex-col gap-3">
+        <% @group.entities.each do |entity| %>
```

Comment:
- [ ] Kindly ensure that the expenses for a group are rendered in the order of the most recent.